#goal definition
float32 range
---
#result definition
bool success
---
#feedback
bool is_target
